modules/ztp-creating-the-siteconfig-custom-resources.adoc

```diff
@@ -83,6 +84,8 @@ spec:
                 next-hop-address: 2620:52:0:10e7::fc
                 table-id: 254
 ----
+<1> If you are using `UEFI SecureBoot`, add this line to prevent failures
```

Comment:
This hard wrapping (which we shouldn't be). Do you mind unwrapping it before I merge?